### DISCUSSION TITLE
Publication of MIMIC Demo Data

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1696] [MIMIC-General] [answered]
Dear MIMIC Maintainers, 

I have an inquriy concerning the publication of MIMIC data. We developed a disease detection software and plan to publicise it. In the development process, we wanted to use openly accessible real-world data for validation of our software. We used data from the MIMIC-Demo database to let physicians generate labels whether a patient had a certain disease or not. Afterwards, we let our software determine the same labels and compared their performance. In order to increase transperancy, we want to make the data, including the "human" and "machine" labels, publicly available. This of course includes publication of the data we extracted from the MIMIC Demo dataset, which we understand is licensed under the Open Data Commons Open Database License, which permits such a use. 

Nevertheless we wanted to absolutely make sure you approve of our process and whether you have any additional request we can acknowledge. Our proposed process looks this:
- publication of the data via our local open data service of our library, including a DOI number
- referencing the DOI of the MIMIC publication as source reference for our data
- submission of the software-paper, referencing both our data, as well as all of the required MIMIC publications

We are thankful for your work enabling our development and research and hope to be able to contribute to MIMIC in the future.

Kind regards

**Comment by aegis301** [comment]
Note: of course we would also release our data under the same license.

**Comment by tompollard** [comment, marked as answer]
@aegis301 Thanks for checking in with us. Everything sounds in line with our requirements, so please go ahead. 

We would like to avoid giving the impression that the full MIMIC dataset can be shared openly, so where possible please take all steps to emphasize this point. 

(e.g. please include statements emphasizing that this is demo data, and that the MIMIC dataset requires (1) training in human research, (2) an identity check, and (3) signing of a Data Use Agreement that prohibits sharing).

**Reply by aegis301** [reply]
Thank you so much for your input we will be happy to include your suggestions on this in the methods section of the paper, as well as in the description of the data. We already established the difference between MIMIC and the demo data on several points in the paper but we will absolutely reiterate on this.
